docs/release_notes/v0.10.0.md

```diff
@@ -0,0 +1,457 @@
+* [Improved Dashboard with new features](https://github.com/dapr/dashboard)
+* New CLI commands (view root cert expiry, export TLS certs, init on k8s with a custom namespace)
+* New components: [InfluxDB binding](https://github.com/dapr/docs/blob/master/reference/specs/bindings/influxdb.md), [Non-interactive OAuth2 middleware](https://github.com/dapr/docs/tree/master/howto/authorization-with-oauth#define-a-client-credentials-grant-component), [Azure Blob Storage state store](https://github.com/dapr/docs/blob/master/howto/setup-state-store/setup-azure-blobstorage.md)
+
+We would like to extend our thanks to all new and existing contributors who helped make this release happen.
+
+If you're new to Dapr, visit the [getting started](https://github.com/dapr/docs/tree/master/getting-started) page and familiarize yourself with Dapr.
+
+Docs have been updated with all the new features and changes of this release. To get started with new capabilities introduced in this release, go to the [Concepts](https://github.com/dapr/docs/tree/master/concepts) and the [How To section](https://github.com/dapr/docs/tree/master/howto#how-tos).
+
+> **Note: This release contains a few [breaking changes](#breaking-changes).**
+
+See [this](#upgrading-to-dapr-0.10.0) section on upgrading Dapr to version 0.10.0.
+
+## New in this release
+
+### Dapr Runtime
+
+**Refactored** Kubernetes operator with Kubebuilder, allowing for multi-replica reconciliation (Thanks @yittg)
+
+**Added** Windows containers support
+
+**Added** support for multiple pub/sub components
+
+**Added** Bulk Get API
+
+**Changed** command line arguments
+
+**Added** stdout log exporter (Thanks @brendandburns)
+
+**Fixed** actors stability bug under high RPS
+
+**Added** Windows e2e tests
+
+**Removed** outdated third-party vendor docs (Thanks @yittg)
+
+**Fixed** actor reminder firing twice bug
+
+**Addded** topic field to Pub/Sub cloud events spec (Thanks @allymparker)
+
+**Upgraded** to use Go 1.15
+
+**Removed** glog dependency and redundant flags
+
+**Fixed** race condition in actor timers
+
+**Updated** dashboard permissions Helm chart
+
+**Fixed** test default values for `TARGET_OS` and `TARGET_ARCH`
+
+**Added** CPU/Memory utilization for performance tests
+
+**Added** ability to turn off telemetry for performance tests
+
+**Fixed** sidecar not resolving secrets after first time installation
+
+**Removed** state retry options
+
+**Fixed** actor reminders overwrite bug
+
+**Added** Grafana dashboards as part of release assets
+
+**Fixed** `grpc_client_method` and `grpc_client_status` metrics tags always empty bug (Thanks @kirk91)
+
+**Added** log entry on sidecar when API auth token is enabled
+
+**Added** versioning scheme to component definition
+
+**Updated** pub/sub to CloudEvents 1.0 spec
+
+**Added** High availability option to Helm Chart
+
+**Fixed** operator k8s service update bug (Thanks @yittg)
+
+**Added** traceparent header in gRPC call
+
+### Dapr CLI
```

Comment:
Breaking change ... dapr cli will install daprd within `$HOME/.dapr/bin` folder. It will not be exported in the Path by default.

Reply:
What's the Windows path?

Reply:
%USERPROFILE%\.dapr\bin

Reply:
Changed